compute/admin_guide/install/install_defender/install_app_embedded_defender_fargate.adoc

```diff
@@ -209,6 +209,24 @@ Prisma Cloud supports the following task definition formats:
     }
 }
 ----
+*Example of the equivalent task definition as a CloudFormation template in YAML:*
+
+----
+Type: AWS::ECS::TaskDefinition
+Properties:
+  ContainerDefinitions:
+  - Name: web
+    Image: nginx
+    EntryPoint:
+    - "/http_server"
+  Cpu: 256
+  ExecutionRoleArn: arn:aws:iam::112233445566:role/ecsTaskExecutionRole
+  Family: webserver
+  Memory: 512
+  NetworkMode: awsvpc
+  RequiresCompatibilities:
+  - FARGATE
+----
 
 [NOTE]
 ====
```

Comment:
@ssugandh this NOTE is no longer relevant
In this feature, except for adding the option for a YAML format, we are also allowing the customer to set up a complete CloudFormation template (including `Resources`, `taskDefinition`, etc,).
So this note should be removed, and we also need to explain somewhere in the article that the user can enter either just the task definition part, or the entire cloudformation template.
We should also provide an example of a full cloudformation template (in JSON + YAML)

Reply:
@grevach : I've added the examples as suggested.